rationale/message_block_layout.md

```diff
@@ -46,4 +46,9 @@ The last piece of the puzzle is determining _which_ row the message is placed at
 
 ### Caveats
 
-The message placement rules described above conflict with the first rule that shares must be ordered by namespace ID, as shares between two messages that are not placed adjacent to each other do not have a natural namespace they belong to. This is resolved by having special transactions the block producer includes that specify a range a zero-padding shares for a given namespace ID (which must be between the namespace IDs of the surrounding real shares, inclusive).
+The message placement rules described above conflict with the first rule that shares must be ordered by namespace ID, as shares between two messages that are not placed adjacent to each other do not have a natural namespace they belong to. This is resolved with some rules:
+1. the namespace ID for message shares must be "even" (i.e. the lowest bit must be `0`), and
+1. the namespace ID for shares between messages must be exactly `1` more than the namespace ID of the preceding message (i.e. must be identical save for the lowest bit being `1` instead of `0`), and.
```

Comment:
Likewise here. It doesn't seem to make sense to say "shares between messages", as messages are shares.

Reply:
Reverted in 6d9648a as per https://github.com/lazyledger/lazyledger-specs/pull/89#discussion_r529009826.